### DISCUSSION TITLE
Implementation curiosity, error codes

### DISCUSSION THREAD

**Discussion by rluvaton** [mobxjs/mobx#3114] [Q&A] [answered]
https://github.com/mobxjs/mobx/blob/8a1ff856043d59396f623f8ca209822b1331d85f/packages/mobx/src/errors.ts#L1-L2

Out of curiosity why did you choose to use error numbers instead of error names?

IMO error names would help understanding the context when reading the source code...

**Comment by urugator** [comment, marked as answer]
Well the point of error codes is to reduce bundle size ... so the more descriptive error codes you have the less bytes you save? 
Dunno what the current tooling capabilities are. The original issue is here https://github.com/mobxjs/mobx/issues/653
